util/src/main/java/org/killbill/billing/util/config/JaxrsConfig.java

```diff
@@ -34,4 +34,9 @@
     @Description("Total timeout for all callables associated to a given api call (parallel mode)")
     TimeSpan getJaxrsTimeout();
 
+    @Config("org.killbill.jaxrs.return.path.like.url")
+    @Default("false")
+    @Description("Type of return for the jaxrs response location")
+    boolean getJaxrsReturnPathLikeUrl();
```

Comment:
Nit: the naming / description is a bit confusing (not obvious it's a boolean). Maybe something more explicit like `isJaxrsLocationFullUrl`?